src/sync_session.cpp

```diff
@@ -125,4 +128,18 @@ RLM_API uint64_t realm_dart_sync_session_register_connection_state_change_callba
 
 } // anonymous namespace
 
+RLM_API void realm_syncsession_report_error_for_testing(realm_sync_session_t* session, uint32_t category, int errorCode, bool isFatal) noexcept
```

Comment:
```suggestion
RLM_API void realm_dart_syncsession_report_error_for_testing(realm_sync_session_t* session, uint32_t category, int errorCode, bool isFatal) noexcept
```

Reply:
Done in https://github.com/realm/realm-dart/pull/577